index.html

```diff
@@ -2,36 +2,70 @@
 
 <head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
   <title>Tic Tac Toe</title>
+  <link href="/styles/style.css" rel="stylesheet" />
 </head>
 
 <body>
-  <h2>Tic Tac Toe</h2>
-  <div class="board">
-    <div id="row-1">
-      <div id="cell-0"></div>
-      <div id="cell-1"></div>
-      <div id="cell-2"></div>
+  <!----Main Wrapper that holds game and status-->
+  <!----player Status bar ------>
+  <div id="main">
+    <h2 class='game-title'>Tic Tac Toe </h2>
+    <h4 id="statusBar">Enjoy The Game</h4>
+    <!-----timer with hr, mins and secs label--->
+    <div class="timer">
+      <div id='clock'><label id="hours">00</label><label id="hours">:</label>
+        <label id="minutes">00</label><label id="minutes">:</label>
+        <label id="seconds">00</label>
+      </div>
     </div>
-    <div id="row-2">
-      <div id="cell-3"></div>
-      <div id="cell-4"></div>
-      <div id="cell-5"></div>
+    <!---create game board cells  -->
+    <div class="board">
+      <div id="row-1">
+        <div class="cell" id="c0"></div>
+        <div class="cell" id="c1"></div>
+        <div class="cell" id="c2"></div>
+      </div>
+      <div id="row-2">
+        <div class="cell" id="c3"></div>
+        <div class="cell" id="c4"></div>
+        <div class="cell" id="c5"></div>
+      </div>
+      <div id="row-3">
+        <div class="cell" id="c6"></div>
+        <div class="cell" id="c7"></div>
+        <div class="cell" id="c8"></div>
+      </div>
     </div>
-    <div id="row-3">
-      <div id="cell-6"></div>
-      <div id="cell-7"></div>
-      <div id="cell-8"></div>
+    <!---buttons to start the game and also enter player names-->
+    <div id="forms-buttons">
+      <section class="game-status">
+        <form id="player-names">
+          <label class='player-name'><input id='player1' type='text' value='' placeholder='Player 1'></label>
+          <section id="start-submit">
+            <button id="start" value="submit" type="button">Start Game</button>
+          </section>
+          <label class='player-name'><input id='player2' type='text' value='' placeholder='Player 2'></label>
+        </form>
+      </section>
+      <!----buttons to pick pVp or pVComp game and also reset the game board when game ends-->
+      <div id="reset-wrapper">
+        <div id="player-wrapper">
+          <button id="playerVPlayer" class="player-computer">Player vs. Player</button>
+        </div>
+        <div class="reset-button">
+          <button id="reset" type="button">Reset Board</button>
+        </div>
+        <div id="computer-wrapper">
+          <button id="playerVComputer" class="player-computer">Player vs. Computer</button>
+        </div>
+      </div>
```

Comment:
The board and buttons all look great. Good use of flexboxes to create the board and page layout!